source/_patterns/02-organisms/50-borrower-dashboard/05-auto-repayments.mustache

```diff
@@ -31,14 +31,17 @@
 				<div style="float: right; margin-left: 1rem;">
 					<a class="auto-repayment-pause">Pause</a>
 				</div>
-				{{#nextRepayment}}
-					Next payment of <b>${{amount}}</b> will be automatically
-					repaid on <b>{{date}}</b>
-				{{/nextRepayment}}
 				{{#preDisbursal}}
 					Enrolled in automatic repayments. Your first repayment
 					will be scheduled once your loan funds are sent to you.
 				{{/preDisbursal}}
+				{{#nextScheduledAutoRepayment}}
```

Comment:
👍 Because we do not have the use case where a pre-disbursal loan would be on a flexible AR plan (and thus potentially have a next scheduled auto repayment), these 3 cases here are mutually exclusive.

Reply:
@ebalcomb Let me make sure I understand how these templates work correctly. Let's assume that we have a pre-disbursal loan and that the borrower has signed up for auto repayments on expected. There will be no scheduled repayments at that time. So preDisbursal will be true, whereas nextScheduledAutoRepayment will be null. Will the template display both the block under {{#preDisbursal}} and the block under {{^nextScheduledAutoRepayment}}?

Reply:
@leoperry yah that's right, and I'm realizing that we really only ever want to show any repaymnet info once they've had disbursal, so I went ahead and wrapped all the autorepayment info in the not-pre-disbursal block.